README.md

```diff
@@ -129,7 +137,7 @@ You don't have to ever use `eject`. The curated feature set is suitable for smal
 
 ### Deployment
 
-No deployment yet.
+[Netlify](https://app.netlify.com/).
```

Comment:
- [ ] The live demo link is opening another link instead of the demo website so I think you should fix it ✅. Furnunetly I found the link from the video presentation 🙂

Reply:
Thus is not the link to Live demo. This is the link to the tool Netlify I used for deployment. The ive demo link is at the top of the README file:

## Live Demo

[Live demo]([./src/images/snapshot_home.jpg](https://stocks-market-data.netlify.app/))

## Description